### PR TITLE
OP-1623 Charlie 7

### PR DESCRIPTION
**Pull request by sacherjj**
Updating configuration genesis files for Charlie-7.
Correcting errors in readme file.
Incrementing version of software.

### CODE REVIEW

**Comment by sacherjj** [comment]
bors try

**Comment by sacherjj** [comment]
bors -try

**Comment by sacherjj** [comment]
My plan is to pull the test .deb and test on our servers prior to merging.  Then tag after merge.

File: resources/production/accounts.csv [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+0188ea4749619a58f7d40f05b3512eddbf48354b440f9154f84417115e3a7b188f,1000000000000000,0
+0123bf1bd107d82bb42f4a92fc1ce19251f341802e1ecfa6ed88a3c7bbbdc302d1,1000000000000000,0
+01c3e4895db6460c68d30076bd4391fc72fe2150d6ec77ec716f51ea2687b9f7bd,1000000000000000,0
+01313a3192db07c50758138e9637faf0358081c2c8709f8a72372685baeaf2a8b0,1000000000000000,0
+0144118517b5ba7e7ca917d5a418c5fb8bf49e279427435d6597f59f8c5bf9ff1f,1000000000000000,0
+01aaa7deb9ec99d6bed089fc938cac4a77bc9539f53556af057f9351a1d3c81de9,1000000000000000,0
+011117189c666f81c5160cd610ee383dc9b2d0361f004934754d39752eedc64957,1000000000000000,0
+01ae2af99944b0b9f92fccf425aa51894ebbad0f4e8e42c66a71dcb999a3bd94ed,1000000000000000,0
+01bdda22721033ae556e92ccf1034f8d14f24c3794fa7befaa831bcae98f9f1e9b,1000000000000000,0
+015ff157d012383e99a3ae93bed0586a23383eed41b0058e63f48ed7b82b067c34,1000000000000000,0
```
Comment (goral09) [inline resources/production/accounts.csv:15]: I think it's too generous to give 14 wallets 10_000_000_000 more tokens that our nodes are bonded with. If they bond with everything they will take over all rounds basically.

Comment (sacherjj) [reply resources/production/accounts.csv:15]: We can give requirements for bonding amounts.  We also want enough funds to run tons of transfers or deploys in the future.

Comment (goral09) [reply resources/production/accounts.csv:15]: We should then increase the amount that other validators are bonded with. This way others can bid but won't take over the network.

**Pull request merged**